### PR TITLE
add Save file+Load save

### PR DESCRIPTION
**Pull request by Mathilbzel**

### CODE REVIEW

File: dialogueMaker/dialogueMaker.cpp [hunk 1]
```diff
@@ -12,17 +11,10 @@ void Scene::printScene() {
...
-    std::cout << dialogue[i];
-    Sleep(1);
-  }
-  std::cout << '\n';
-  std::cout << "---------------------------------------------------------------------------------------------------------------------------";
-  std::cout << '\n';
+  std::cout << "---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------";
+  std::cout << '\n' << dialogue << '\n';
+  std::cout << "---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------";
+  std::cout << '\n';\
```
Comment (OrneryCash) [inline dialogueMaker/dialogueMaker.cpp]: Please remove a leftover backslash
```suggestion
  std::cout << '\n';
```

File: dialogueMaker/dialogueMaker.cpp [hunk 2]
```diff
@@ -116,17 +108,12 @@ void Game::addCurrentEvent(std::string event) {
 void Game::askForChoice() {
   while (1) {
     std::string choice;
-    std::string x = "Enter Your Choice: (q to quit game): ";
-    for (int i = 0; i < x[i]; i++)
-    {
-      std::cout << x[i];
-      Sleep(15);
-    }
-//    std::cout << "\n\nEnter your choice: ";
```
Comment (OrneryCash) [inline dialogueMaker/dialogueMaker.cpp]: So, no animation for this anymore?

File: example.cpp [hunk 3]
```diff
@@ -58,6 +59,6 @@ int main() {
   });
 
   // Game loop
-  Game::runGame("begin");
+  Game::LoadSave("save.txt");
```
Comment (OrneryCash) [inline example.cpp:62]: If a player plays this for the first time, does the player have a save.txt file from the start?

**Review by OrneryCash — CHANGES_REQUESTED** [review]
Aside from the leftover backslash, it's just a discussion.

File: dialogueMaker/dialogueMaker.cpp (hunk 2 above)
Comment (Mathilbzel) [reply dialogueMaker/dialogueMaker.cpp]: accidently deleted, I will bring it back asap

File: example.cpp (hunk 3 above)
Comment (Mathilbzel) [reply example.cpp:62]: Forget to add save.txt file, but at the start there will be start stats and begin scene

Comment (OrneryCash) [reply example.cpp:62]: Okayy

**Review by OrneryCash — APPROVED** [review]
LGTM.

**Comment by Mathilbzel** [comment]
> LGTM.

Merge?

**Comment by OrneryCash** [comment]
> > LGTM.
> 
> Merge?

Yes, please

**Comment by Mathilbzel** [comment]
Merge attempt failed
Head branch is out of date. Review and try the merge again.
How to solve?

**Comment by OrneryCash** [comment]
> Head branch is out of date. Review and try the merge again.

Enter this in the terminal:
`git pull origin master`
`git push origin Me2253`

**Comment by Mathilbzel** [comment]
> git pull origin master

Cannot pull it said:
fatal: couldn't find remote ref master

**Comment by OrneryCash** [comment]
> Cannot pull it said: fatal: couldn't find remote ref master

Sorry it has to be main instead of master

**Pull request merged**